client/analytics/ad-tracking.js

```diff
@@ -181,6 +210,12 @@ module.exports = {
 		nextFunction();
 	},
 
+	recordAddToCart: function( cartItem ) {
```

Comment:
There doesn't seem to be a good reason for a second declaration of `recordAddToCart` or a wrapper for `recordPurchase` in the export. I'll update it per your question.